doc/MANUAL.md

```diff
@@ -99,7 +107,7 @@ For a quick overview of usage and how it works, you can also see [the readme fil
 ## How it works
 
 This scanner looks at the strings found in binary files to see if they
-match vulnerable versions of a small set of popular open source libraries.
+match vulnerable versions of a small set of popular open-source libraries.
```

Comment:
We should be saying "open source" with a space rather than a dash.  (I know, I've seen grammar checkers say otherwise, but convention is definitely in favour of the space over the dash.